firestore/tests/unit/test_watch.py

```diff
@@ -403,7 +403,7 @@ class DummyRemove(object):
 
         remove = DummyRemove()
         proto.document_remove = remove
-        proto.document_delete = None
+        proto.document_delete = ""
```

Comment:
These changes seem to push us over the line to use a real `ListenResponse` message, e.g.:

```python
def _make_listen_response(**kw):
    from google.cloud.firestore_v1beta1.proto.firestore_pb2 import ListenResponse
    return ListenResponse(**kw)
```

because the messages get all their scalar fields iniitalized to defaults for free.

Reply:
I suppose we could, but the proto object isn't the only composite object that needs to be created; it needs a target_change object and a document_change object as attributes and those are also complex objects, so we'd either have to pass a textproto to  ListenResponse.FromString or invent some other deserializer to do that and I'm not sure doing either of those things would make the code any clearer.